tests/test_filters.py

```diff
@@ -118,3 +121,18 @@ def test_capitalize_first_for_non_strings():
     assert capitalize_first([{'list': 'of'}, 'things']) == [{'list': 'of'}, 'Things']
     assert capitalize_first({'this': 'thing'}) == {'this': 'thing'}
     assert capitalize_first('https://www.example.com') == 'https://www.example.com'
+
+
+@pytest.mark.parametrize("bool", (False, True,))  # Shouldn't matter
+def test_preserve_line_breaks(bool):
```

Comment:
I think the idea is to test functional equivalency rather than having any idea about treating Markup differently. 

If that's already a given, then, yeah, we don't need both cases.